tests/healthchecker/healthchecker.mocks.ts

```diff
@@ -110,4 +110,26 @@ export const cenarios: HealthCheckDetailedTestCenario = {
       port: 8001,
     },
   },
+  databaseIntegrationTruthy: {
+    expected: true,
+    config: {
+      type: HealthTypes.Database,
+      name: "jest-test-postgres",
+      host: "localhost",
+      dbPort: 5432,
+      dbName: "postgres",
+      dbUser: "postgres",
+      dbPwd: "root",
+      dbDialect: "postgres",
```

Comment:
maybe the database dialects could be mapped in an enum into the interface!

Reply:
Done!